config/setupcfg.py

```diff
@@ -72,6 +74,18 @@ def setup_logging(name, level):
     logging.setup(CONF, name)
     LOG = logging.getLogger(None)
 
+    # Add option to do Log Rotation
```

Comment:
On Line 75, can we do?
LOG = logging.getLogger(__name__)

Reply:
Will check

Reply:
Only 'None' works. In which ever classes / modules this LOG is used there __name__ is used for initializing the logger.